src/client/mtna/archive-template.jade

```diff
@@ -19,6 +19,7 @@ div.relative(role="main"
     md-card(
       ng-if="state.pre.length > 0"
       flex-gt-sm="90" flex-gt-md="80"
+      class="firefox-tweak"
```

Comment:
It's for md cards that have one of those flex props set. Particularly flex-gt-md